var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -294,3 +294,41 @@
     #
     - { key: 'AutoUpdate.Enabled', value: 'n' }
   when: platform == "azure"
+
+#
+# Customize the GCP linux environment.
+#
+# Update the override file for the GCP instance. This file gets
+# applied dynamically by running google_instance_setup script.
+#
+- blockinfile:
+    path: /etc/default/instance_configs.cfg.template
+    create: yes
+    block: |
+      #
+      # Disable the accounts daemon to prevent adding/removing
+      # users on the engine.
+      #
+      [Daemons]
+      accounts_daemon = false
+
+      #
+      # Disable user supplied startup/shutdown scripts from running on
+      # the engine.
+      #
+      [MetadataScripts]
+      shutdown = false
+      startup = false
+  when:
+    - platform == "gcp"
+    - not ansible_is_chroot
```

Comment:
why is this needed? we should be able to customize the file in the chroot just fine, right? and we still won't run the gcp command, since the handler has the "when" clause, right?

I'd rather use "ansible_is_chroot" as little as possible, and I don't think we need it here.

Reply:
Yes, we can remove it here and just prevent the handler from running.